framework/src/Volo.Abp.Ddd.Domain/Volo/Abp/Domain/Entities/Caching/AbstractKeyMultiTenancyEntityCache.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Volo.Abp.Caching;
+using Volo.Abp.Domain.Repositories;
+using Volo.Abp.MultiTenancy;
+
+namespace Volo.Abp.Domain.Entities.Caching
+{
+    public abstract class AbstractKeyMultiTenancyEntityCache<TEntity, TCacheItem, TKey> :
```

Comment:
No need to such a class. I think this is needed to add tenantid to the cache key. Instead of such a class, this can be done by an if condition (like `if TEntity is IMultiTenant`) in the `AbstractKeyEntityCache.GetCacheKey` method.